### DISCUSSION TITLE
What version of fastai, fastcore, torch do i need for this?

### DISCUSSION THREAD

**Discussion by rekil156** [KeremTurgutlu/self_supervised#39] [Q&A] [answered]
Hi Kerem, this has been super helpful in understanding the SimCLR for me. 
Is there a place where we can see what version of fastai, fastcore needs to be installed? I'm experimenting on colab. 
I'm ending up with - ModuleNotFoundError: No module named ‘fastai.vision.all’ .
Thanks

**Comment by KeremTurgutlu** [comment, marked as answer]
Hi I am glad it has been useful to you :) You can simply:

pip install self-supervised

Which will install the dependencies for you such as fastai, timm and kornia. The library has been tested with fastai 2.2.8 most recently. As of today it uses the latest version of all the libraries.

**Reply by KeremTurgutlu** [reply]
Fastai will install fastcore and torch as it's dependecies.

**Reply by rekil156** [reply]
AWESOME :) 
Thanks Kerem. 
On colab had to download the editable version of fastai. Colab seems to not have updated to fastai version 2

**Reply by KeremTurgutlu** [reply]
Yeah colab has an old version, simply do `pip install -U fastai` if on colab.
